### PR TITLE
releases/1.25: Add Jeremy and Joseph as Branch Manager shadows

### PR DESCRIPTION
**Pull request by justaugustus**
#### What type of PR is this:

/kind documentation

#### What this PR does / why we need it:

In discussion with the other @kubernetes/sig-release-leads, I'd like to propose @jeremyrickard and @jrsapi as 1.25 Branch Manager shadows.

They have worked in/around the Release Team for several cycles, are highly trusted members of SIG Release (both having served in the Emeritus Adviser role for the Release Team), and have the technical background to thrive in this role.

This roster addition would implicitly make them Release Manager Associates, which implied by the action of opening this PR, I am in favor of. :)

We'd like to workshop ideas for release manager selection, but let's treat that as a trailing item, to be discussed as part of https://github.com/kubernetes/sig-release/pull/1958.

Signed-off-by: Stephen Augustus <foo@auggie.dev>

Assigning a few people for approval...
SIG Release leads:
/assign @saschagrunert @cpanato @puerco

1.25 Branch Manager:
/assign @Verolop

1.25 Release Team lead:
/assign @cici37

cc: @kubernetes/release-engineering @kubernetes/release-team-leads

/hold
the cycle is well underway, so I will consider this mergeable with an `/lgtm` from:
- branch manager
- RT lead
- one of other SIG Release leads

### CODE REVIEW

**Comment by palnabarun** [comment]
+100 🙌🏽 

/lgtm

**Comment by ameukam** [comment]
/lgtm
Welcome to the band!

**Comment by puerco** [comment]
Super deluxe team :rocket: 
/lgtm

**Comment by cici37** [comment]
/lgtm
This is awesome!!!!!

**Comment by xmudrii** [comment]
/lgtm
What an awesome team! 🚀

**Review by saschagrunert — APPROVED** [review]

**Comment by justaugustus** [comment]
We've got the requisite approvals, so removing the hold:
/hold cancel

@jeremyrickard and @jrsapi -- welcome to the team!!

Your first assignment!
Work w/ @Verolop to run through the onboarding checklist in https://github.com/kubernetes/sig-release/issues/1965 :)

**Pull request merged**
